docs/android/tips/release.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Build for release"
+sidebar_label: "Build for release"
+---
+
+:::warning
+Ensure the keystore is securely stored and not included in version control. It also applies for the credentials.
+:::
+
+## Building for publishing
```

Comment:
This was the title on the README. We can change it to Building for release instead maybe.

Reply:
"building for release" rolls off the tongue easier :)

Reply:
Adjusted